src/TitleDetails/TitleDetails.js

```diff
@@ -118,7 +118,7 @@ const TitleDetails = ({
   poLine,
   title,
   onUnreceive,
-  vendorsMap,
+  vendorsMap = {},
```

Comment:
I don't think it should have a default value exactly here since `vendorsMap` is the required object in the prop types

Reply:
Not sure. It looks like something has changed. The page is failing on the Snapshot.